cli/enry/main.go

```diff
@@ -136,6 +139,50 @@ func printPercents(out map[string][]string, buff *bytes.Buffer) {
 	}
 }
 
+func printFile(file string, buff *bytes.Buffer) {
```

Comment:
I think there are some issues with this function as it is (which are not completely independent):
* I don't think `printFile`, as it is, is the proper name for this function since it doesn't print anything.
* Why doesn't it just print the file as promised instead of handling a buffer and doing the printing outside?
* Why is a pointer to a buffer a function parameter instead of just being a return value?